powerfulseal/policy/ps-schema.yaml

```diff
@@ -216,6 +224,14 @@ definitions:
               - "$ref": "#/definitions/filterDayTime"
               - "$ref": "#/definitions/filterRandomSample"
               - "$ref": "#/definitions/filterProbability"
+          retries:
+            type: array
```

Comment:
The appears to only take the first element of the array - in which case, should this become an object, rather than an array? Is there a use case for having multiple retry strategies?

Reply:
There is not a reason I could think of that an array of retries would be correct. I'll update it to an object instead of an array